content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+Eviction is the process by which {{<glossary_tooltip text="pods" term_id="pod" length="short">}}
+in a {{<glossary_tooltip text="node" term_id="node" length="short">}} are
+terminated. 
+
+In Kubernetes, there are two types of pod eviction:
+
+  1. Graceful eviction
+  2. Out-of-resource eviction ({{<glossary_tooltip text="kubelet" term_id="kubelet" length="short">}}
```

Comment:
There are 2 types of out-of-resource eviction:

- The first is more like node resource-pressure eviction, acted on by the kubelet.
- The second is cluster level out-of-resource preemption, which clears space from any node to be able to schedule higher priority pods. This is acted on by scheduler.

Reply:
What's the difference between eviction and preemption? We have a separate topic talking about preemption. 

For this I can add: 
There are two types of out-of-resource eviction: 
* Node-level eviction: managed by the kubelet and triggered by resource pressure in nodes
* Cluster-level preemption: managed by the scheduler and triggered to clear space on nodes to schedule higher priority pods. For more information, see [Pod Priority and Preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#preemption).

Reply:
lgtm, but maybe @sftim has a better phrasing.

Reply:
@sftim - I updated the glossary entry for out-of-resource eviction. I'll work it into the doc in a way that makes sense and get you to look at it on my next push.